browser/ui/brave_ads/ad_notification_popup.cc

```diff
@@ -51,21 +51,19 @@ namespace {
 // AdNotificationPopup management to NotificationPopupCollection
 std::map<std::string, AdNotificationPopup*> g_ad_notification_popups;
 
-const int kShadowElevation = 5;
+constexpr int kShadowElevation = 5;
 
 constexpr SkColor kLightModeBackgroundColor = SkColorSetRGB(0xed, 0xf0, 0xf2);
 constexpr SkColor kDarkModeBackgroundColor = SkColorSetRGB(0x20, 0x23, 0x27);
 
 constexpr SkColor kLightModeBorderColor = SkColorSetRGB(0xd5, 0xdb, 0xe2);
 constexpr SkColor kDarkModeBorderColor = SkColorSetRGB(0x3f, 0x41, 0x45);
-const int kBorderThickness = 1;
+constexpr int kBorderThickness = 1;
 
-#if defined(OS_WIN)
-const int kCornerRadius = 0;
+#if defined(OS_WIN) || defined(OS_LINUX)
+constexpr int kCornerRadius = 0;
```

Comment:
Does Linux not support a corner radius with shadows?

Reply:
No, setting the radius for widget has no effect.
If set the radius for background, it leads to folllowing artefacts:

![Screenshot from 2021-07-02 20-31-47](https://user-images.githubusercontent.com/86598290/124282157-b4be6d00-db74-11eb-9b87-121303a5e884.png)